scpd/lms/templates/head-extra.html

```diff
@@ -0,0 +1,10 @@
+## mako
+
+<%namespace name='static' file='/static_content.html'/>
```

Comment:
Per convention in the other files here,
add an empty line between the `mako` imports and `HTML` tags.